zebra-consensus/src/block.rs

```diff
@@ -22,6 +22,7 @@ use tower::{Service, ServiceExt};
 use zebra_chain::block::{self, Block};
 use zebra_state as zs;
 
+use crate::errors::*;
```

Comment:
Calling the module `error` rather than `errors` would be more consistent with the other modules.

Reply:
Thanks, changed this at https://github.com/ZcashFoundation/zebra/pull/1070/commits/676dce1d7632df9c242fe0c64cfc12f60f347017